project/profiles/capacity4more/behat/features/contact.feature

```diff
@@ -22,4 +22,4 @@ Feature: Contact
     Given I am logged in as user "alfrednobel"
     When I go to user "mariecurie" contact form
     Then I should see "Contact"
-    And I see the wysiwyg
+    And I should see the "#cke_edit-c4m-contact-user-body-value" element
```

Comment:
@DavidHernandez 
The previous test passed even without the wysiwyg, hence I had to change it.

I've tested the current one 4 times while removing the wysiwyg and putting it back.